client/my-sites/domains/domain-management/transfer/transfer-out/select-ips-tag.jsx

```diff
@@ -0,0 +1,215 @@
+		}
+
+		this.setState( {
+			selectedRegistrar,
+			showDialog: true,
+		} );
+	}
+
+	onCloseDialog = ( action ) => {
+		if ( 'submit' === action ) {
+			this.handleSubmit();
+		}
+
+		this.setState( { showDialog: false } );
+	}
+
+	renderGoToGainingRegistrar() {
+		const { translate } = this.props;
+
+		return (
+			<Card
+				className="transfer-out__select-ips-tag"
+				href="https://example.com"
+				target="_blank"
+				highlight="success"
+			>
+				{ translate(
+					'Success! Click here to go to your new registrar and complete the transfer.'
+				) }
+			</Card>
+		);
+	}
+
+	renderIpsTagSelect() {
+		const { selectedDomainName, translate } = this.props;
+
+		return (
+			<div>
+				<p>
+					{ translate(
+						'Please enter the IPS tag of the registrar you wish to transfer ' +
+						'{{strong}}%(selectedDomainName)s{{/strong}} to.',
+						{ args: { selectedDomainName }, components: { strong: <strong /> } }
+					) }
+				</p>
+
+				<SearchCard
+					autoFocus
+					disableAutocorrect
+					onSearch={ this.handleSearchInputChange }
+					onBlur={ this.hideSuggestions }
+					onKeyDown={ this.handleKeyDown }
+					placeholder="Start typing an IPS tag..."
+					value={ this.state.ipsTagInput }
+				/>
+				<Suggestions
+					ref={ this.setSuggestionsRef }
+					query={ this.state.currentQuery }
+					suggestions={ this.getSuggestions() }
+					suggest={ this.handleSuggestionClick }
+				/>
+				<FormButton onClick={ this.popOverDialog } >
+					{ this.props.translate( 'Submit' ) }
+				</FormButton>
+
+				{ this.renderDialog() }
+			</div>
+		);
+	}
+
+	renderDialog() {
+		const { selectedDomainName, translate } = this.props;
+		const { selectedRegistrar } = this.state;
+		const buttons = [
+			{ action: 'cancel', label: translate( 'Cancel' ) },
+			{ action: 'submit', label: translate( 'Yes, Submit!' ), isPrimary: true },
+		];
+
+		return (
+			<Dialog
+				isVisible={ this.state.showDialog }
+				buttons={ buttons }
+				onClose={ this.onCloseDialog }>
+				<h1>{ translate( 'Transfer Confirmation' ) }</h1>
+				<p>
+					{ translate( 'Please verify you wish to set the registrar for ' +
+						'{{strong}}%(selectedDomainName)s{{/strong}} to the following:',
+						{ args: { selectedDomainName }, components: { strong: <strong /> } }
+					) }
+				</p>
+				<p>
+					<strong>{ selectedRegistrar.tag }</strong>&nbsp;&nbsp;
+					{ selectedRegistrar.registrarName ? '(' + selectedRegistrar.registrarName + ')' : '' }
+				</p>
+				<p>
+					{ translate( 'After submitting this tag change, the domain will no longer be in our system. ' +
+						'You will need to contact the gaining registrar to complete the transfer and regain ' +
+						'control of the domain.'
+					) }
+				</p>
+			</Dialog>
+		);
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<div>
+				<SectionHeader label={ translate( 'Transfer Domain To Another Registrar' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Transfer to another registrar' )` *ES Score: 12*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/uk-transfer-out)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).